flight/Libraries/inc/uavtalk_priv.h

```diff
@@ -103,6 +103,8 @@ typedef struct {
 #define UAVTALK_TYPE_OBJ_ACK   (UAVTALK_TYPE_VER | 0x02)
 #define UAVTALK_TYPE_ACK       (UAVTALK_TYPE_VER | 0x03)
 #define UAVTALK_TYPE_NACK      (UAVTALK_TYPE_VER | 0x04)
+#define UAVTALK_TYPE_FILEREQ   (UAVTALK_TYPE_VER | 0x10)
```

Comment:
I also had something like this in mind. Probably doesn't really fit the commit description though. 🙃